lib/wrappers/python/emulator_contourwall.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import cv2
+import time
+
+class EmulatorContourWall:
+    def __init__(self):
+        self.__cv_window_name = "ContourWall Emulation"
+        self.rows = 40
+        self.cols = 60
+        self.cell_size = 10
+        self.pushed_frames: int = 0
+        self.matrix = np.zeros((self.rows * self.cell_size, self.cols * self.cell_size, 3), dtype=np.uint8) 
+        self.pixels: np.ndarray = np.zeros((self.rows, self.cols, 3), dtype=np.uint8)
```

Comment:
The actual ContourWall library has a field `pushed_frames`, make sure to add that here too.

Everytime `EmulatorContourWall.show()` is called you increase the field by one.